sharktuner/sharktuner/libtuner.py

```diff
@@ -835,6 +847,8 @@ def generate_candidate_specs(
                 candidate_id=candidate_num,
                 spec_path=spec_path,
                 td_spec_str=td_spec_str,
+                knob_assignment=knob if candidate_num != 0 else None,
+                # No knob_assignment for baseline.
```

Comment:
We put comments either above the things we describe or on the RHS (if they fit the same line)
```suggestion
                # No knob_assignment for baseline.
                knob_assignment=knob if candidate_num != 0 else None,
```